patterns/proxy/mod.ts

```diff
@@ -0,0 +1,59 @@
+import { types } from "polkadot_dev/mod.ts"
+import { Proxy } from "polkadot_dev/mod.ts"
+import { Event } from "polkadot_dev/types/pallet_proxy/pallet.ts"
+import { RuntimeEvent } from "polkadot_dev/types/polkadot_runtime.ts"
+import { hex, MultiAddress, Rune, RunicArgs } from "../../mod.ts"
+
+export function addProxy<X>(
+  ...[proxy, delegate]: RunicArgs<X, [proxy: MultiAddress, delegate: MultiAddress]>
```

Comment:
Perhaps we'd like to turn these into a Rune subclass (`Rune<Uint8Array /* proxy accountId */>`) as to enable the following?

```ts
const proxy = Rune.constant(multiaddr).into(ProxyRune)

await proxy
  .add(alice.address)
  .chain(() => proxy.remove(bob.address))
  .run()
```

Reply:
Perhaps even a `transfer` method which encapsulates the two steps.

```ts
await proxy
  .transfer(bob.address)
  .signed({ sender: alice })
  .sent()
  .finalized()
  .run()
```

Reply:
It wouldn't be possible (given that there are multiple signers involved). This is a constraint of the utility pallet's batching methods, not this design.